pages/glossary/accessible-name.md

```diff
@@ -1,6 +1,8 @@
 ---
 title: Accessible Name
 key: accessible-name
+unambiguous: true
```

Comment:
Technically, "accessible name" **is** ambiguous because the algorithm doesn't explain which kind of whitespace should be trimmed.
However, we seem to always have considered it as unambiguous… So we kept it that way…
(it is used in the Applicability or Expectation of a dozen rules so far…)

Reply:
Presume the same ASCII Whitespace as defined elsewhere

Reply:
The problem is that the algorithm uses "whitespace" without more precision (and different browsers interpret that differently).
And we are not in control of the spec of that algorithm…
(the issue has been raised: https://github.com/w3c/accname/issues/55)